### ISSUE TITLE
update batch node to support 1.8 and 2.0

### ISSUE THREAD

**Issue by mongojarle** [mblackstock/node-red-contrib-influxdb#93]
I would like to write several measurements to the database in one operation. Is there any plans to update the batch node to allow for use with InfluxDB 2.x?

**Comment by mblackstock** [comment]
The output node can write multiple points as documented in the README "if msg.payload is an array of arrays, it will be written as a series of points containing fields and tags."  There is an example there.  Also see the section [Multiple Points](https://mblackstock.com/2021/02/07/influxdb-node.html) in the blog post.

The batch node was implemented to address a request to format the data as in the underlying node.js [influx library version 5.x](https://www.npmjs.com/package/influx).  Since this library only supports InfluxDb 1.x or earlier, it will be deprecated at some point, there is no plan to update it.

**Comment by mongojarle** [comment]
My problem is that the output node can only write to one measurement. I would like to write to several measurements, like I did in the batch node.

**Comment by mblackstock** [comment]
I see - I misunderstood what you needed (multiple measurements, not just points).  Let me look into it!

**Comment by araisch** [comment]
I would really appreciate that feature as well, due to the fact that Influx2 provides some awesome features, but it is not really nice to scatter measurements arriving in one array into hundreds of messages.
@mblackstock A big thank you for maintaining this anyway.

**Comment by mblackstock** [comment]
This is available for testing on the branch `batch-node-update`.  You can test it using:

```
npm install git://github.com/mblackstock/node-red-contrib-influxdb#batch-node-update
```

Let me know if it works for you.  You can also specify integers using strings in the batch node for 1.8 and 2.0 configurations as in the output node.

**Comment by mongojarle** [comment]
Mike, thank you very much. I have tested a little bit, and as far as I can see, it is working the same as for influx1.8.
Excellent work! Helped me a great deal towards migrating to Influx2.0!

**Comment by mblackstock** [comment]
fixed in version 0.6.0
